src/ai/ai_help_structs.cc

```diff
@@ -31,9 +31,13 @@
 
 namespace AI {
 
-constexpr int kNoAiTrainingMutation = 200;
-constexpr int kUpperDefaultMutationLimit = 150;
-constexpr int kLowerDefaultMutationLimit = 75;
+constexpr int kNoAiTrainingMutation = 501;
```

Comment:
I Believe this might be the culprit for the testsuite failure in conjunction with line 773 and 776.

Reply:
Yes, but I want to fix it in different way